src/brs/db/sql/Db.java

```diff
@@ -128,7 +128,7 @@ private static DSLContext getDSLContext() {
     if (con == null) {
       return DSL.using(databaseInstance.getDataSource(), dialect, settings);
     } else {
-      settings.setStatementType(StatementType.STATIC_STATEMENT);
+      settings.setStatementType(StatementType.PREPARED_STATEMENT);
```

Comment:
yes.. good move... we can even omit this line as per default PREPARED are being used. I think STATIC was overseen when Harry was working on something experimental - I removed that line completely in #885